### DISCUSSION TITLE
Notification when cat health falls below certain level

### DISCUSSION THREAD

**Discussion by Nicka244** [runelite/runelite#15062] [Ideas]
I was hoping someone could implement a notification for cat health when they are farming spices. This would help people who lose their cats while trying to farm spices for stews and boosts.

**Comment by patriq** [comment]
If no one would mind I could try to implement this :)

EDIT: Upon further investigation, I came to the conclusion that cat health doesn't work like when Summoning was added to the game. There used to be (and I believe there still is in RS3) a varpbit one could mask to obtain the cat's current growth and health. 
Currently the only way I can think of doing this would be to possibly check for game messages that say the cat is either lonely or hungry. Unless someone has better ideas?

**Comment by StephanBijzitter** [comment]
@patriqdesigns the overhead chat lines from a car represent information as well. It's far from perfect, but everything combined could make it pretty accurate.

>Text above the kitten can also indicate its status; a long "Meeeooowww!" will mean it is very hungry, for example, while the same drawn out meow but with a "..." indicates it is feeling lonely.

http://oldschoolrunescape.wikia.com/wiki/Cat

**Comment by patriq** [comment]
Yeah I have been raising a cat since yesterday to see what I could possibly
use.
I still have to check how can I check for overhead chat lines, but I am
unaware of how would I map it to a number that states how hungry the cat is.
Maybe the notification could just show up when he is really hungry or
lonely?

On Mon, May 28, 2018 at 3:51 PM, Stephan Bijzitter <notifications@github.com
